core/src/test/java/io/paradoxical/cassieq/unittests/SlowTests.java

```diff
@@ -98,10 +110,11 @@ public FailingWorker(
                 final QueueName queueName,
                 final Collection<Integer> collection,
                 final ExecutorService executorService) {
-            super(client, queueName, collection, executorService);
+            super(client, queueName, collection, executorService, null);
```

Comment:
its a reference to the clock tick. i was just lazy since it doesn't need to tick here. we only tick on success and the bad worker isn't ever going to tick